python/packages/ai/teams/ai/prompts/message.py

```diff
@@ -96,9 +96,9 @@ class Citation:
     url: str
     filepath: str
 
-
+dataclass_json
```

Comment:
missing @ symbol at the beginning

Reply:
Were you able to successfully run this bot locally and test?

Reply:
Sorry, I didn't test the code. While I don't mind furnishing an example using personal time, I cannot justify spending company Azure resources for testing. I hope you can understand.

This example follows the corresponding JS sample and is very minimal. Hopefully you can easily test it on your end.

Thanks!

Reply:
Apologize for the confusion. I tested the code for a company internal project and it was working properly. I didn't test the example that I was asked to provided.

Reply:
The missing @ symbol has been added.

Reply:
@lifan0127 Yes of course, no worries- we can test on our end. Thanks for verifying it via your internal project!